src/boot_fake_node/mod.rs

```diff
@@ -475,5 +486,9 @@ pub async fn execute(
         handle.await.unwrap();
     }
 
+    if let Ok(mut child) = anvil_process {
+        child.kill().unwrap();
+    }
```

Comment:
This needs to be brought into the `run_tests/cleanup.rs` logic to improve likelihood of cleanup (e.g. if user sends SIGINT via Ctrl+C). Some codelines to look at (we can also chat about this if you like):

https://github.com/kinode-dao/kit/blob/4129e1c954b6d240b866ade0d346cdda5917bb05/src/run_tests/cleanup.rs
https://github.com/kinode-dao/kit/blob/4129e1c954b6d240b866ade0d346cdda5917bb05/src/boot_fake_node/mod.rs#L407-L428
https://github.com/kinode-dao/kit/blob/4129e1c954b6d240b866ade0d346cdda5917bb05/src/run_tests/types.rs#L69-L73

Reply:
added the anvil process to the NodeCleanupInfo struct in [e9815ab](https://github.com/kinode-dao/kit/pull/140/commits/e9815ab714d02ea56897099b6a31e123ad87aea4), also kit chain now works in a similar manner.